src/TreeHouse/BehatCommon/AbstractDoctrineOrmContext.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+namespace TreeHouse\BehatCommon;
+
+use Behat\Gherkin\Node\TableNode;
+use Behat\Symfony2Extension\Context\KernelAwareContext;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Doctrine\Common\Util\Inflector;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use PHPUnit_Framework_Assert as Assert;
+
+abstract class AbstractDoctrineOrmContext extends AbstractPersistanceContext implements KernelAwareContext
+{
+    use KernelAwareTrait;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * @var string
+     */
+    private $defaultPrefix;
+
+    /**
+     * @param EntityManagerInterface|null $entity_manager
+     * @param string                      $default_prefix
+     */
+    public function __construct(
+        EntityManagerInterface $entity_manager = null,
+        $default_prefix = 'AppBundle'
+    ) {
+        $this->entityManager = $entity_manager;
+        $this->defaultPrefix = $default_prefix;
+    }
+
+    /**
+     * @Given /^the following ((?!.*should).*) exist(s)?:$/
```

Comment:
Is het niet mogelijk toevallig om dit in de interface op te nemen, zodat je het niet 2 keer hoeft te definieren?